modules/ROOT/pages/execution-plans/operators.adoc

```diff
@@ -2772,6 +2772,58 @@ Batch size 128
 Total database accesses: 1, total allocated memory: 200
 ----
 
+[[query-plan-repeat]]
+== Repeat (Trail)
+// Repeat(Trail)
+
+Given a start node, the `Repeat(Trail)` operator will traverse quantified path patterns that cannot be solved with the xref::execution-plans/operators.adoc#query-plan-varlength-expand-all[`VarLengthExpand(All)`] operator
+
+.Repeat(Trail)
+======
+
+.Query
+[source, cypher]
+----
+PROFILE
+MATCH (me:Person) ((a)-[:FRIENDS_WITH]-(b) WHERE a.name <> b.name){1,2} (friend:Person)
+RETURN me, friend
+----
+
+.Query Plan
+[role="queryplan", subs="attributes+"]
+----
+Planner COST
+
+Runtime PIPELINED
+
+Runtime version {neo4j-version-minor}
+
+Batch size 128
+
++------------------+----+-----------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| Operator         | Id | Details                                             | Estimated Rows | Rows | DB Hits | Memory (Bytes) | Page Cache Hits/Misses | Time (ms) | Pipeline            |
++------------------+----+-----------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| +ProduceResults  |  0 | me, friend                                          |             15 |   48 |       0 |                |                        |           |                     |
+| |                +----+-----------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| +Filter          |  1 | friend:Person                                       |             15 |   48 |      96 |                |                        |           |                     |
+| |                +----+-----------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| +NullifyMetadata |  7 |                                                     |             15 |   48 |       0 |                |                        |           |                     |
+| |                +----+-----------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| +Repeat(Trail)   |  2 | (me) (...){1, 2} (friend)                           |             15 |   48 |       0 |          16784 |                    0/0 |     1.146 | Fused in Pipeline 2 |
+| |\               +----+-----------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| | +Filter        |  3 | not a.name = b.name AND isRepeatTrailUnique(anon_2) |              5 |   48 |     216 |                |                        |           |                     |
```

Comment:
Should we also explain the cardinalities displayed for `Repeat` or is that still too much in flux / too internal?

Reply:
We don't do that for any other operator, so I would not start doing that for the most complex operator we have :laughing: